Description: Stop unnecessary linkage against -lgcrypt and -lgpg-error.
 Neither of these libraries are used by libetpan, see
 "grep -Erli 'gpg_err|gcry'".
 This version of the patch is just required to generate the orig tarball
 from upstream git tag, hence has been removed from series file.
Author: Andreas Metzler <ametzler@debian.org>, Ricardo Mones <mones@debian.org>
Origin: vendor
Last-Update: 2014-11-24

diff --git a/configure.ac b/configure.ac
index 3fcea52..1e1cb82 100644
--- a/configure.ac
+++ b/configure.ac
@@ -240,7 +240,7 @@ fi
 if test "x$with_gnutls" = "xyes"; then
   AC_DEFINE([USE_GNUTLS],1, [Define to use GnuTLS])
   AC_DEFINE([USE_SSL], 1, [Define to use OpenSSL])
-  GNUTLSLIB="-lgnutls -lgcrypt -lgpg-error -lz"
+  GNUTLSLIB="-lgnutls"
 else
   GNUTLSLIB=""
 fi
From 945f3702182a7b7920c440512c15cce23ad1bacd Mon Sep 17 00:00:00 2001
From: Po-Chuan Hsieh <sunpoet@sunpoet.net>
Date: Sun, 4 Nov 2018 01:52:05 +0800
Subject: [PATCH] Fix build with curl 7.62.0 (#311)

from CHANGES:
ssl: deprecate CURLE_SSL_CACERT in favour of a unified error code
Long live CURLE_PEER_FAILED_VERIFICATION
---
 src/low-level/feed/newsfeed.c | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/src/low-level/feed/newsfeed.c b/src/low-level/feed/newsfeed.c
index 97e292b7..13472801 100644
--- a/src/low-level/feed/newsfeed.c
+++ b/src/low-level/feed/newsfeed.c
@@ -535,7 +535,9 @@ static int curl_error_convert(int curl_res)
   case CURLE_SSL_ENGINE_SETFAILED:
   case CURLE_SSL_CERTPROBLEM:
   case CURLE_SSL_CIPHER:
+#if LIBCURL_VERSION_NUM < 0x073e00
   case CURLE_SSL_CACERT:
+#endif
   case CURLE_FTP_SSL_FAILED:
   case CURLE_SSL_ENGINE_INITFAILED:
     return NEWSFEED_ERROR_SSL;
